#!/usr/bin/env python3
PACKAGE = "simple_sim_wheel_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('max_steer', double_t, 0, 'Max Steering Angle', 0.4, 0.0, 0.4)

gen.add('max_speed', double_t, 0, 'Max Velocity', 3, 0, 5)

gen.add('max_accel', double_t, 0, 'Max Acceleration', 0.02, 0, 0.1)

gen.add('max_decel', double_t, 0, 'Max Acceleration', 0.05, 0, 0.1)



exit(gen.generate(PACKAGE, "wheel_drive", "SimWheel"))